wrappers/python/demo/test.py

```diff
@@ -25,250 +25,251 @@
 entropy     = "entropy"
 rev_idx = 1
 
-schema = Schema.create("schema name", "schema version", issuer_id, ["name","age","sex","height"])
-
-cred_def_pub, cred_def_priv, cred_def_correctness = CredentialDefinition.create(
-    schema_id,
-    schema,
-    issuer_id,
-    "tag",
-    "CL",
-    support_revocation=True
-)
-
-(rev_reg_def_pub, rev_reg_def_private) = RevocationRegistryDefinition.create(
-    cred_def_id,
-    cred_def_pub,
-    issuer_id,
-    "some_tag",
-    "CL_ACCUM",
-    10
-)
-
-time_create_rev_status_list = 12
-revocation_status_list = RevocationStatusList.create(
-    rev_reg_id,
-    rev_reg_def_pub,
-    issuer_id,
-    time_create_rev_status_list,
-    True
-)
-
-link_secret = LinkSecret.create()
-link_secret_id = "default"
-
-cred_offer = CredentialOffer.create(
-    schema_id,
-    cred_def_id,
-    cred_def_correctness
-)
-
-cred_request, cred_request_metadata = CredentialRequest.create(
-    entropy,
-    None,
-    cred_def_pub,
-    link_secret,
-    link_secret_id,
-    cred_offer
-)
-
-issue_cred = Credential.create(
-    cred_def_pub,
-    cred_def_priv,
-    cred_offer,
-    cred_request,
-    {
-        "sex": "male",
-        "name": "Alex",
-        "height": "175",
-        "age": "28"
-    },
-    None,
-    rev_reg_id,
-    revocation_status_list,
-    CredentialRevocationConfig(
+def run():
```

Comment:
Nothing changed in this file except it being in a `run` function. Which I did for it to be able to be run with `poetry run demo` which I thought was nice. Maybe there is a nicer way or a more idiomatic way. Maybe with `__main__`?